feg/cloud/go/services/feg_relay/gw_to_feg_relay/tests/relay_test.go

```diff
@@ -113,6 +245,9 @@ func TestNHRouting(t *testing.T) {
 	// Register FeG's test Hello Server
 	feg_protos.RegisterHelloServer(srv.GrpcServer, &testHelloServer{})
 
+	// Register FeG's Central Session Controller Server
+	sessionProxy := &testCentralSessionController{resultChan: make(chan string, 2)}
```

Comment:
shouldnt you have to have two `testCentralSessionController`

Reply:
not added as explained below